UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Services/ProductDataService.cs

```diff
@@ -68,21 +88,82 @@ public async Task<ExchangeSetServiceResponse> CreateProductDataByProductIdentifi
             return response;
         }
 
+        private int GetFilesize(SalesCatalogueProductResponse salesCatalogueResponse)
+        {
+            int fileSizeCount = 0;
+            if (salesCatalogueResponse != null && salesCatalogueResponse.ProductCounts.ReturnedProductCount > 0)
+            {
+                foreach (var item in salesCatalogueResponse.Products)
+                {
+                    fileSizeCount += item.FileSize.Value;
+                }
+            }
+            return fileSizeCount;
+        }
+        public bool IsAzureAdB2CSource(AzureAdB2C azureAdB2C)
+        {
+            bool isAzureB2CUser = false;
+            string B2CAuthority = $"{azureAdB2CConfiguration.Value.Instance}{azureAdB2CConfiguration.Value.TenantId}/v2.0/";// for B2C Token
+            string AdB2CAuthority = $"{azureAdConfiguration.Value.MicrosoftOnlineLoginUrl}{azureAdB2CConfiguration.Value.TenantId}/v2.0";// for AdB2C Token
+            string Audience = azureAdB2CConfiguration.Value.ClientId;
+            if (azureAdB2C.IssToken == B2CAuthority && azureAdB2C.AudToken == Audience)
+            {
+                isAzureB2CUser = true;
+            }
+            else if (azureAdB2C.IssToken == AdB2CAuthority && azureAdB2C.AudToken == Audience)
+            {
+                isAzureB2CUser = true;
+            }
+            return isAzureB2CUser;
+        }
+        public ExchangeSetServiceResponse CheckIfExchangeServiceSetFileTooLarge(SalesCatalogueResponse salesCatalogueResponse)
+        {
+            int fileSize = GetFilesize(salesCatalogueResponse.ResponseBody);
+            var FileSizeInMB = CommonHelper.ConvertBytesToMegabytes((long)fileSize);
```

Comment:
change variable name to fileSizeInMB. Change method name from CheckIfExchangeServiceSetFileTooLarge to CheckIfExchangeSetTooLarge